### DISCUSSION TITLE
Canvas or DOM for UI when using the WASM target?

### DISCUSSION THREAD

**Discussion by martin-braun** [DioxusLabs/dioxus#758] [Q&A] [answered]
A quick question: When compiling towards the WASM, does the output produce a DOM (like ReactNative) with a JS bridge to access it, or does it produce a canvas that it will be drawn on (like Flutter)?

On online demo of a project that runs in the web would've been helpful, but there is none.

Thanks in advance. 🙏

**Comment by ealmloff** [comment, marked as answer]
Dioxus renders using the dom. For an online demo, the [Dioxus website](https://dioxuslabs.com) is built useing dioxus

**Reply by martin-braun** [reply]
This is awesome, thank you! 😀
